app/src/main/kotlin/com/intive/picover/main/navigation/NavControllerHolder.kt

```diff
@@ -0,0 +1,27 @@
+package com.intive.picover.main.navigation
+
+import android.annotation.SuppressLint
+import androidx.navigation.NavHostController
+
+// TODO temporary navController holder, to be removed after migration to voyager is fully done
+object NavControllerHolder {
```

Comment:
This is just temporary code, until the main screen is migrated to voyager and navcontroller is not needed anymore, because we will use navigator from voyager. Not super proud of it but I don't want to invest too much time into some overengineered solution which will be removed soon 😄